### PR TITLE
Issue #11 Fixed

### PR DESCRIPTION
**Pull request by BraxtonElmer**
Fixes issue #11 
The code in Digital_guj_scraper.py now web scrapes the link and sees the notifications, and sends a mail to users and saves all of the current notifications in the notifications.json file, and when it is run again, the data is compared with notifications.json and if there is new notifications in the web, it is again mailed to the user and then the new notifications are saved in the notifications.json file.

### CODE REVIEW

File: Digital_guj_scraper.py [hunk 1]
```diff
@@ -63,9 +31,7 @@ def Sendemail(date,receivers_email):
     msg['Subject'] = 'Latest GTU Notification'
 
     # Create the body of the message (a plain-text and an HTML version).
-    # html = (date + '\n'+Notification+ '\n'+link)
-    html = (date )
-
+    html = (date + '\n'+Notification+ '\n'+link)
```
Comment (Rjchauhan18) [inline Digital_guj_scraper.py]: Fix this 

image

**Review by Rjchauhan18 — CHANGES_REQUESTED** [review]

**Comment by BraxtonElmer** [comment]
@Rjchauhan18 The error you mentioned has been fixed in my latest commit. I have also done the required testing to ensure that the code functions properly.

File: Digital_guj_scraper.py [hunk 2]
```diff
@@ -13,12 +13,11 @@
 
 
 # smtp connection
-def Sendemail(date,Notification, link,receivers_email):
+def Sendemail(date,Notification, link):
```
Comment (Rjchauhan18) [inline Digital_guj_scraper.py:15]: Can you please improve mail text by removing this unwanted texts.

image

File: Digital_guj_scraper.py [hunk 3]
```diff
@@ -13,12 +13,11 @@
...
   # Define email addresses to use
-    sender_email = os.environ.get('SMTP_SENDER_EMAIL')#sender email
+    sender_email =os.environ.get('SMTP_SENDER_EMAIL')#sender email
```
Comment (Rjchauhan18) [inline Digital_guj_scraper.py:18]: please import dotenv module for getting this variables from .env file

**Comment by Rjchauhan18** [comment]
Your are going in the right direction just made these small changes .

File: Digital_guj_scraper.py [hunk 4]
```diff
@@ -1,56 +1,23 @@
-import os
-import requests 
+import requests
+import json
 from bs4 import BeautifulSoup
+import os
+from os import path
+import pickle
```
Comment (Rjchauhan18) [inline Digital_guj_scraper.py]: remove the modules which are not use and also update requirements.txt if any new module is added.

File: Digital_guj_scraper.py (hunk 2 above)
Comment (BraxtonElmer) [reply Digital_guj_scraper.py:15]: hello, can you please explain what you mean by unnecessary text in this image? All of them are present in the notification part of the webpage

**Comment by BraxtonElmer** [comment]
I have added dotenv library and fixed env variables loading now, and added dotenv to requirements. I have also fixed the typo in env variable name in readme and added a variable for receiver email.

**Pull request merged**